idea_town/frontend/static-src/app/views/experiment-page.js

```diff
@@ -215,20 +218,28 @@ export default PageView.extend({
   renderUninstallSurvey(evt) {
     evt.preventDefault();
     // TODO: Hardcoded survey, for now. Populate via API later?
-    this.renderSubview(new FeedbackView({
-      id: 'disabled-feedback',
-      experiment: this.model.url,
-      title: 'Why are you stopping?',
-      questions: [
-        { value: 'broken', title: 'This thing is broken!' },
-        { value: 'dislike', title: 'I don\'t like this feature.' },
-        { value: 'notuseful', title: 'This isn\'t useful for me.' },
-        { value: 'other', title: 'Something else.' }
-      ],
-      onSubmit: () => {
-        this.uninstall(this.updateAddon, this.model);
-      }
-    }), 'body');
+    const l10nIDs = [
+      'feedbackUninstallTitle', 'feedbackUninstallAnswerBroken',
+      'feedbackUninstallAnswerDislike', 'feedbackUninstallAnswerNotUseful',
+      'feedbackUninstallAnswerOther'
+    ];
+    document.l10n.formatValues(...l10nIDs).then((results) => {
+      const strings = zipObject(l10nIDs, results);
+      this.renderSubview(new FeedbackView({
+        id: 'disabled-feedback',
+        experiment: this.model.url,
+        title: strings.feedbackUninstallTitle,
+        questions: [
+          { value: 'broken', title: strings.feedbackUninstallAnswerBroken },
+          { value: 'dislike', title: strings.feedbackUninstallAnswerDislike },
+          { value: 'notuseful', title: strings.feedbackUninstallAnswerNotUseful },
+          { value: 'other', title: strings.feedbackUninstallAnswerOther }
+        ],
```

Comment:
why do you prefer to mass-l10n using `formatValues` rather than build DOM with `data-l10n-id`?